test_scripts/API/VehicleData/WindowStatus/013_App_version_is_equal_to_windowStatus_param_version_is_listed_in_DB_and_API.lua

```diff
@@ -0,0 +1,48 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0261-New-vehicle-data-WindowStatus.md
+--
+-- Description: Check that SDL successfully processes VD RPC with new `windowStatus` param in case app version is equal to 6.0
```

Comment:
```suggestion
-- Description: Check that SDL successfully processes VehicleData RPCs with new `windowStatus` param in case app version is equal to 6.0
```